snippets/snippets.json

```diff
@@ -4,16 +4,16 @@
     "body": [
       "import 'package:over_react/over_react.dart';",
       "",
-      "part '${1:FileName}.over_react.g.dart';",
+      "part '${TM_FILENAME_BASE}.over_react.g.dart'; // ignore: uri_has_not_been_generated",
```

Comment:
Why were these comments added? Most of our consumers use workiva_analysis_options to ignore this instead of a comment, so I don't think we should include the comment by default.

Reply:
Because I always forget about that 😄  Updated!